support/cas-server-support-pm-webflow/src/test/java/org/apereo/cas/pm/web/flow/actions/SendForgotUsernameInstructionsActionEmailMessageBodyTextTests.java

```diff
@@ -0,0 +1,155 @@
+package org.apereo.cas.pm.web.flow.actions;
+
+import lombok.val;
+import org.apereo.cas.notifications.CommunicationsManager;
+import org.apereo.cas.util.junit.EnabledIfPortOpen;
+import org.apereo.cas.web.flow.CasWebflowConstants;
+import org.apereo.cas.web.support.WebUtils;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.binding.message.MessageContext;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Import;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.mock.web.MockServletContext;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.webflow.context.servlet.ServletExternalContext;
+import org.springframework.webflow.core.collection.LocalAttributeMap;
+import org.springframework.webflow.execution.Action;
+import org.springframework.webflow.execution.RequestContext;
+import org.springframework.webflow.test.MockParameterMap;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+/**
+ * This is {@link SendForgotUsernameInstructionsActionEmailMessageBodyTextTests}.
+ *
+ * @author Misagh Moayyed
+ * @since 6.3.0
+ */
+@EnabledIfPortOpen(port = 25000)
+public class SendForgotUsernameInstructionsActionEmailMessageBodyTextTests {
```

Comment:
Remove the test class. Merge everything into SendForgotUsernameInstructionsActionEmailMessageBodyTests.

Reply:
Done